### PR TITLE
Add Dictionary to Env while creating a dictionary

### PR DESCRIPTION
**Pull request by ankurtrivedi**

### CODE REVIEW

File: src/main/resources/synthetic.xml [hunk 1]
```diff
@@ -201,6 +201,8 @@
                   description="XML with the fields to set on the CI"/>
         <property name="addToEnvironment" category="input" kind="boolean" required="false"
                   description="Switch to decide if to add the CI to an environment"/>
+        <property name="addDictToEnvironment" category="input" kind="boolean" required="false"
```
Comment (jdewinne) [inline src/main/resources/synthetic.xml:204]: Wouldn't it make more sense to reuse the `addToEnvironment` and automatically detect if the created CI is a container or a dict?

File: src/main/resources/xlr_xldeploy/createCITask.py [hunk 2]
```diff
@@ -19,3 +19,5 @@
     xld_client.create_ci(ciID, ciType)
 if addToEnvironment:
     xld_client.add_ci_to_environment(envID, ciID)
+if addDictToEnvironment:
```
Comment (jdewinne) [inline src/main/resources/xlr_xldeploy/createCITask.py:22]: Can be removed if `addToEnvironment` is reused.

**Review by jdewinne — CHANGES_REQUESTED** [review]
You can also fix the license issue by running a `licenseFormatMain` or `licenseFormatTest`

**Comment by ankurtrivedi** [comment]
Will add another pull request with modifications as suggested.

**Pull request closed without merging**